handlers/follower.js

```diff
@@ -174,7 +158,7 @@ follower.post('/users/:id/followers', authentication,
     validate([param('id').isInt({ min: 1 })]),
     async (request, response) => {
         try {
-            const follower = await followerController.followUp(request.params.id, request.token);
+            const follower = await followerController.followUp(request.params.id, request.authorization.user.id);
```

Comment:
It's not clear here, you are passing id from params and user id from authentication. Based on the endpoint I would expect them to be the same